encoding.h

```diff
@@ -168,11 +168,14 @@
 #define MSTATEEN0_CS       0x00000001
 #define MSTATEEN0_FCSR     0x00000002
 #define MSTATEEN0_JVT      0x00000004
+#define MSTATEEN0_PRIV14   0x0080000000000000
```

Comment:
I think 114 would be a little clearer.

Do we also need to reserve the corresponding bit in `hstateen`?  I think the answer is "no, we can cross that bridge later", but wanted to make sure you agree and that it wasn't an accidental omission.

Reply:
Updated to PRIV114.

I did not reserve the bit in `hstateen0` yet as there was not a state defined.